src/pages/AboutPage.js

```diff
@@ -45,6 +45,34 @@ class AboutPage extends React.Component {
                         land and pay our respects to their culture, their
                         people and elders past and present.
                     </p>
+                    <a
+                        href={
+                            "mailto:database@infoxchange.org" +
+                            "?subject=" +
+                            encodeURIComponent("Ask Izzy add site/service") +
+                            "&body=" +
+                            encodeURIComponent(`Service provider name:
+
+Service name:
```

Comment:
It would be nicer to have this indented and use a function to trim the leading whitespace on each line. If that's something trivial to implement then it would be good to see, but otherwise ++